### PR TITLE
feature: added support for env TEST_NGINX_RANDOMIZE, which when set allows parallel testing

### PR DESCRIPTION
**Pull request by agentzh**

### CODE REVIEW

File: lib/Test/Nginx/Util.pm [hunk 1]
```diff
@@ -435,7 +442,12 @@ sub add_block_preprocessor(&) {
 #our ($PrevRequest)
 our $PrevConfig;
 
-our $ServRoot   = $ENV{TEST_NGINX_SERVROOT} || File::Spec->catfile(cwd() || '.', 't/servroot');
+our $ServRoot   = $ENV{TEST_NGINX_SERVROOT} || File::Spec->rel2abs('t/servroot');
```
Comment (p0pr0ck5) [inline lib/Test/Nginx/Util.pm]: can probably remove the extra whitespace between the var and the assignment operator, given that we're no longer aligning with the chunk of declarations below us

**Comment by agentzh** [comment]
@p0pr0ck5 removed the extra spaces. Thanks!

File: lib/Test/Nginx/Util.pm [hunk 2]
```diff
@@ -542,6 +554,10 @@ sub kill_process ($$) {
 }
 
 sub cleanup () {
+    if ($Verbose) {
+        warn "cleaning up everyitng";
```
Comment (p0pr0ck5) [inline lib/Test/Nginx/Util.pm]: Pedantic: typo ("everything")

Comment (agentzh) [inline lib/Test/Nginx/Util.pm]: @p0pr0ck5 Fixed. Thanks!

**Comment by doujiang24** [comment]
@agentzh Works fine on side now:)

**Comment by agentzh** [comment]
Merged. Thank you all for the review!

**Pull request closed without merging**